katello/katello-change-hostname

```diff
@@ -50,108 +124,132 @@ def yesno
   end
 end
 
-def hammer_ping_success
-  hammer_ping = `hammer ping`
-  success = $?.success?
-  service_failed = hammer_ping.include? "FAIL"
-  success && !service_failed
+def fail_with_message(message, opt_parser=nil)
+  STDOUT.puts message
+  puts opt_parser if opt_parser
+  exit(false)
 end
 
-options = {}
-options[:program] = DEFAULT_PROGRAM
-options[:scenario] = DEFAULT_SCENARIO
-options[:system_check] = false
+def check_for_certs_tar
+  STDOUT.puts "Checking for certs tarball"
+  if @options[:certs_tar]
+    if File.file?(@options[:certs_tar])
+      true
+    else
+      fail_with_message("#{@options[:certs_tar]} does not exist! Please check the file path and try again")
+    end
+  else
+    fail_with_message("You must specify --certs-tar argument when on a #{@proxy}." \
+                      " These can be generated on the #{DEFAULT_PROGRAM} server using capsule-certs-generate and copied to this machine.")
+  end
+end
 
-opt_parser = OptionParser.new do |opt|
-  opt.banner = "Usage: katello-change-hostname HOSTNAME [OPTIONS]"
-  opt.separator  ""
-  opt.separator  "Example:"
-  opt.separator  " katello-change-hostname foo.example.com -u admin -p changeme"
-  opt.separator  ""
-  opt.separator  "Options"
+def get_fpc_answers
+  fpc_answers = YAML.load_file("/etc/foreman-installer/scenarios.d/#{FOREMAN_PROXY_CONTENT}-answers.yaml")
```

Comment:
I'm not sure I understand why? If I'm getting the foreman-proxy answers, shouldn't I specify that file?